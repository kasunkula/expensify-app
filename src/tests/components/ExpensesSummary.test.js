import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should render summary for 1 expenses', () => {
  const wrapper = shallow(<ExpensesSummary expensesCount={1} expensesTotal={235}/>);
  expect(wrapper).toMatchSnapshot();
});

test('should render summary for multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expensesCount={3} expensesTotal={43343432}/>);
    expect(wrapper).toMatchSnapshot();
 });