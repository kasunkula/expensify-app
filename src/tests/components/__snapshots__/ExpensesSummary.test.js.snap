// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render summary for 1 expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
    , total 
    $0.00
  </h1>
</div>
`;

exports[`should render summary for multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
     
    expenses
    , total 
    $0.00
  </h1>
</div>
`;
